editor.planx.uk/src/@planx/components/List/Public/index.test.tsx

```diff
@@ -39,6 +39,12 @@ describe("Basic UI", () => {
     expect(textInput).toBeInTheDocument();
     expect(textInput.type).toBe("text");
 
+    // Input field descriptions are generated from schema if they exist...
+    const textInputDescription = getByText(
+      /Please make it cute/,
+    ) as HTMLElement;
+    expect(textInputDescription).toBeInTheDocument();
```

Comment:
Descriptions now checked in basic setup test so we proactively catch this next time!